nsv13/code/modules/overmap/weapons/projectiles_fx.dm

```diff
@@ -563,12 +563,31 @@ Misc projectile types, effects, think of this as the special FX file.
 	tracer_type = /obj/effect/projectile/tracer/disabler
 	muzzle_type = /obj/effect/projectile/muzzle/disabler
 	impact_type = /obj/effect/projectile/impact/disabler
+	relay_projectile_type = /obj/item/projectile/beam/laser/heavylaser/phaser/relayed
+
+/obj/item/projectile/beam/laser/heavylaser/phaser/relayed
+	projectile_piercing = PASSGLASS|PASSGRILLE|PASSTABLE
+	damage = 40 //yes, the explosion will probably gib them. but if it doesn't, let's give them a chance to live, instead of smiting them with the full wraith of the Enterprise 
+
+/obj/item/projectile/beam/laser/heavylaser/phaser/relayed/on_hit(atom/target, blocked)
+	. = ..()
+	explosion(get_turf(target), 0, 2, 4, 6, ignorecap = FALSE, flame_range = 4) //I halved the BSA's explosion, and then poked it some myself - big boom
 
 /obj/item/projectile/beam/laser/phaser/pd
 	name = "point defense phaser"
 	damage = 60 // Doesn't scale with power input, but fires fairly quickly especially when upgraded
 	icon = 'nsv13/icons/obj/projectiles_nsv.dmi'
 	icon_state = "pdphaser"
+	flag = "overmap_medium" //won't just ignore armor anymore... 
+	armour_penetration = 50 //half as strong as it used to be, but still powerful
```

Comment:
As the PD is a phaser subtype, it inherits those values over basetype, which means its flag already was `overmap_heavy` as opposed to a base laser one.

Reply:
oh shoot. not heavy, it's medium, but yeah that's my bad.